clean-base.py

```diff
@@ -69,34 +80,35 @@
 # Build file list in which to search for placeholders to replace
 
 for subdir, dirs, files in os.walk("."):
-	for file in files:
-		filepath = subdir + os.sep + file
+    for file in files:
+        filepath = subdir + os.sep + file
 
-		if any([filepath.endswith(ext) for ext in [".py", ".yaml", ".html", ".md", ".sh", ".json", ".js", ".less"]]):
-			file_list.append(filepath)
+        if any([filepath.endswith(ext) for ext in [".py", ".yaml", ".html", ".md", ".sh", ".json", ".js", ".less"]]):
+            file_list.append(filepath)
 
 # Replace placeholders with values entered by user or defaults
 
 for file_obj in file_list:
-	lines = []
-	with open(file_obj) as infile:
-		for line in infile:
-			for src, target in replacements.items():
-				line = line.replace(src, target)
-			lines.append(line)
-	with open(file_obj, "w") as outfile:
-		for line in lines:
-			outfile.write(line)
+    lines = []
+    with open(file_obj) as infile:
+        for line in infile:
+            for src, target in replacements.items():
+                line = line.replace(src, target)
+            lines.append(line)
+    with open(file_obj, "w") as outfile:
+        for line in lines:
+            outfile.write(line)
 
 # Update submodules
 
 if os.path.exists(".git") and clean_history:
-	print("Clean git history")
-	subprocess.check_output("git checkout --orphan main_tmp", shell=True)
-	print(subprocess.check_output("git branch -D main", shell=True).decode().rstrip("\n"))
-	subprocess.check_output("git branch -m main", shell=True)
-	print("Current branch is:", subprocess.check_output("git branch --show-current", shell=True).decode().rstrip("\n"))
-	print("---")
+    print("Clean git history")
+    subprocess.check_output("git checkout --orphan main_tmp", shell=True)
+    print(subprocess.check_output("git branch -D main", shell=True).decode().rstrip("\n"))
+    subprocess.check_output("git branch -m main", shell=True)
+    print(
+        "Current branch is:", subprocess.check_output("git branch --show-current", shell=True).decode().rstrip("\n"))
```

Comment:
This looks very ugly, can we write instead?
```suggestion
	print("Current branch is:",
		  subprocess.check_output("git branch --show-current", shell=True)
		  .decode().rstrip("\n"))
```

Reply:
We could, but the PEP8 131 convention needs to be considered. 
So the compromise i can offer is: 
`print(
        "Current branch is:",
        subprocess.check_output("git branch --show-current", shell=True)
        .decode().rstrip("\n")
    )`

Reply:
anyways i tried to implement a somehow prettier print statement